### ISSUE TITLE
Ability to fully remove item from project

### ISSUE THREAD

**Issue by crutkas** [JaneaSystems/PowerToys-DevProjects#39]
Not just the check to include, i need ability to fully remove item
![image](https://github.com/JaneaSystems/PowerToys-DevProjects/assets/1462282/c5d1dedf-4ede-4a15-9921-9c873115029d)

**Comment by crutkas** [comment]
Think it removes after you did a launch

**Comment by donlaci** [comment]
@plante-msft 
A proposal to consider:
We could change the meaning of the checkboxes. Currently the checkbox controls the included/excluded state of an app.
We could consider all listed apps as part of the project. The checkboxes could remain but we could have some actions with the selected apps.
One could be delete the selected apps. (For now I don't see any other action but there might come some later)
In the top section of the editor page we would need a "select/deselect all" checkbox and a button for the delete action.
The select all checkbox is required in issue #50

**Comment by plante-msft** [comment]
@donlaci I agree with the proposal, and we need Select All option per screen.

**Comment by hamza-usmani** [comment]
+1 to the proposal, makes sense for future-proofing the edit experience as well
